src/helpers/stateHelper.ts

```diff
@@ -36,3 +36,21 @@ export const observableState = <T>(initial: T): ObservableState<T> => {
     set: (newValue: T) => subject$$.next(newValue)
   }
 }
+
+export type TriggerStream = {
+  stream$: Observable<string>
+  trigger: () => void
+}
+
+/**
+ * Helper to create a stream, which can trigger changes of it
+ *
+ * It might be handy to trigger other, depending streams in a queue to do something
+ */
+export const triggerStream = (): TriggerStream => {
+  const subject$$ = new BehaviorSubject('')
```

Comment:
I'm not yet familiar with RxJS. Should empty string be the correct value here?

Reply:
Yep, that's just to provide any data here. It could be anything in this `BehaviorSubject` object as long as it has the same type called in `next()`. This data is hidden for the outside world and it's just used to "trigger" a "change" of the subject (`BehaviorSubject`).